cmd/openshift-install/features.go

```diff
@@ -0,0 +1,27 @@
+package main
+
+import (
+	"fmt"
+
+	"github.com/spf13/cobra"
+)
+
+var hiddenFeatures = [...]string{
+	"terraform-spot-masters",
+}
```

Comment:
I suppose grep works fine here, and perhaps we don't need to over engineer but would it be more convenient to structure this output, say in json or something?

Reply:
@patrickdillon what about a `--json` argument for that?

Reply:
Done

Reply:
Cool; though the current use case still requires:
- parsing the output with something (be it `grep` or `jq` or `[[ "$(...)" == *'"terraform-spot-masters"'* ]]`)
- accounting for what happens when invoking this on an older installer binary (stdout is empty, RC is nonzero)

...so YAGNI for now :shrug:

The only thing I can think of that would make it easier to consume would be something like `openshift-install is-hidden-feature-supported terraform-spot-masters` which exits zero if yes, nonzero if no. That way it can reliably be consumed as:

```bash
if ! openshift-install is-hidden-feature-supported terraform-spot-masters >/dev/null 2>&1; then
    ...
fi
```

...because I'll get a nonzero RC on older installers as well as newer ones that have this code but disclaim the support.

Reply:
to be clear: I'm not asking for :arrow_up:, just thinking aloud :)